modules/compliance-evaluate-kubeletconfig-rules.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+:_content-type: CONCEPT
+[id="compliance-evaluate-kubeletconfig-rules"]
+= Evaluating `KubeletConfig` rules against default configuration values
+
+{product-title} infrastructure may contain incomplete configuration files at run time, and nodes will assume default configuration values for missing configuration options. Some configuration options can be passed as command line arguments. As a result, the Compliance Operator cannot verify if the configuration file on the node is complete, as it may be missing options used in the rule checks.
```

Comment:
> ! may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.
```suggestion
{product-title} infrastructure might contain incomplete configuration files at run time, and nodes assume default configuration values for missing configuration options. Some configuration options can be passed as command line arguments. As a result, the Compliance Operator cannot verify if the configuration file on the node is complete because it might be missing options used in the rule checks.
```